### PR TITLE
 Add mount unittest

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
@leakingtapan @ddebroy please /ok-to-test

**Comment by leakingtapan** [comment]
thx for the PR!
/ok-to-test

**Comment by gliptak** [comment]
@leakingtapan PR is green

File: pkg/driver/mount_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:2]: NIT : 2020

File: pkg/driver/mount_test.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	defer os.RemoveAll(dir)
+
+	targetPath := filepath.Join(dir, "targetdir")
+
+	var (
+		mountObj = newNodeMounter()
+	)
+
+	if mountObj.MakeDir(targetPath) != nil {
+		t.Fatalf("Expect no error but got: %v", err)
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:41]: Could u check the dir also exists on file system?

File: pkg/driver/mount_test.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	}
+	defer os.RemoveAll(dir)
+
+	targetPath := filepath.Join(dir, "targetfile")
+
+	var (
+		mountObj = newNodeMounter()
+	)
+
+	if mountObj.MakeFile(targetPath) != nil {
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:59]: Could u check the file also exists on file system?

File: pkg/driver/mount_test.go (hunk 1 above)
Comment (gliptak) [reply pkg/driver/mount_test.go]: updated

File: pkg/driver/mount_test.go (hunk 2 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:41]: updated

File: pkg/driver/mount_test.go (hunk 3 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:63]: updated

**Comment by gliptak** [comment]
@leakingtapan PR is green

**Pull request merged**